ports/drogon/vcpkg.json

```
@@ -1,10 +1,12 @@
 {
   "name": "drogon",
   "version": "1.8.2",
+  "port-version": 1,
   "description": "A C++14/17 based HTTP web application framework running on Linux/macOS/Unix/Windows",
   "homepage": "https://github.com/an-tao/drogon",
   "documentation": "https://drogon.docsforge.com/master/overview/",
   "license": "MIT",
+  "supports": "!uwp",
```

```suggestion
  "supports": "!uwp",
```
For the downstream of the dependent port `c-ares`, it may have to be changed. 
I will finish it. :)